store/confirmCDS.go

```diff
@@ -0,0 +1,166 @@
+package store
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	log "github.com/sirupsen/logrus"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+
+	"github.com/bitmark-inc/autonomy-api/schema"
+)
+
+type CDSCountryType string
+
+const (
+	CdsUSA         = "United States"
+	CdsUSALevel    = "county"
+	CdsTaiwan      = "Taiwan"
+	CdsTaiwanLevel = "country"
```

Comment:
this variable is not used.

Reply:
I think it will be used soon in next PR. If not, I will remove it